### PR TITLE
php-redis 3.1.4 (except php53)

### PR DESCRIPTION
**Pull request by dasmfm**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by javian** [comment]
@dasmfm can you please submit these as one PR instead of all of these? Its hell to merge them and serve no real purpose for this tap.

**Comment by dasmfm** [comment]
@javian , sure, sorry for mess, it's all bump-formula-pr :(

**Comment by javian** [comment]
Please exclude php53 since we don't have a working bottle of php on that version of the os.

**Comment by javian** [comment]
@dasmfm yeah I'm not mad =) and the regular brew taps should keep that format still its just a pain when updating the same version of the software for so many versions. We need to drop some of these soon.

**Comment by dasmfm** [comment]
@javian , done.

**Comment by javian** [comment]
Its still complaining about
```
==> brew audit homebrew/php/php56-redis --online
==> FAILED
Error: 1 problem in 1 formula
homebrew/php/php56-redis:
  * 'revision 1' should be removed
```
also in 5.5 but I can't see that you left them in there.

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by dasmfm** [comment]
my bad ( fork was out of sync with upstream.

**Comment by dasmfm** [comment]
@BrewTestBot test this please

File: Formula/php55-redis.rb [hunk 1]
```diff
@@ -4,16 +4,15 @@ class Php55Redis < AbstractPhp55Extension
...
   head "https://github.com/phpredis/phpredis.git"
-  revision 1
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "bc5ebaa99924419525fdac24ed9a4fd96dd9bfc2b4578a3e6d3fe8e67ad196f1" => :high_sierra
-    sha256 "629378883ed275f6c2809a8def695fc23890a79e198fcb02172ef888f6ec5f7f" => :sierra
-    sha256 "a4e8e1118048f7f856d210c6ba9cf62e5b7e9dcfa2bfe30ab2d33345ebe104a9" => :el_capitan
+    sha256 "d06ff4ae34a4d954698a46c71885d265997f54df7f89061db0c4502e311dde9e" => :sierra
+    sha256 "10843cb178110b42d16a9531c668cfe0d6df35b785365de9dc63ebe3d2e395cc" => :el_capitan
```
Comment (SMillerDev) [inline Formula/php55-redis.rb]: No need to change the bottles here

File: Formula/php56-redis.rb [hunk 2]
```diff
@@ -4,16 +4,15 @@ class Php56Redis < AbstractPhp56Extension
...
   head "https://github.com/phpredis/phpredis.git"
-  revision 1
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "f5ec9ab1515598169df6ea76e550d024bf18192c21dcc50defebf7c3dbddc08e" => :high_sierra
-    sha256 "e55bb35c436a180fd12b0be9393938ebc62968fcc56f03913f1b3b59318ee18d" => :sierra
-    sha256 "95c86f3d58f56f0330672680261791c0130ada5e1e6a506e9f025ba6ee683445" => :el_capitan
+    sha256 "55c35605f6c661c89ec1099b8981395170f1727dfe0022f868ab7f87d17d554f" => :sierra
+    sha256 "be8fcfb87ef0e464b26a6fee3a813478384158ee518951f4e08996e667570a52" => :el_capitan
```
Comment (SMillerDev) [inline Formula/php56-redis.rb]: here neither

**Comment by dasmfm** [comment]
@SMillerDev you're right, fixed.

**Comment by SMillerDev** [comment]
Thanks @dasmfm

**Pull request closed without merging**